.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,4 +6,7 @@ contact_links:
   - name: Report an issue with a lint
     url: https://github.com/dart-lang/linter/issues/new?assignees=&labels=&template=issue-report.md&title=
     about: Create an issue report for an existing lint.
+  - name: Propose a lint for inclusion in Flutter
+    url: https://github.com/flutter/flutter/issues/new?assignees=&labels=p:%20flutter_lints
```

Comment:
I think we should track those issues here. We want the two sets of lints to be aligned, no?

Reply:
Totally.  I guess I was thinking that folks who wanted to discuss Flutter-specific lints (adding on top of recommended) might take that conversation to the Flutter repo but perhaps it's best to at least start the conversation here.  And maybe this is too nuanced anyway?

@goderbauer: wdyt?

Reply:
Starting the conversation right here in the lints repository sounds good to me.